content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+
+```json
+{
+  "data": {
+    "attributes": {
+      "enabled": true,
+      "idp-cert": "SAMPLE-CERTIFICATE",
+      "slo-target": "https://example.com/slo",
+      "sso-target": "https://example.com/sso",
+      "attr-groups": "MemberOf",
+      "attr-site-admin": "SiteAdmin",
+      "site-admin-role": "site-admins",
+      "sso-api-token-session-timeout": 1209600
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/admin/settings/saml
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id":"saml",
+    "type":"settings/saml",
+    "attributes": {
+      "enabled": true,
+      "idp-cert": "SAMPLE-CERTIFICATE",
+      "slo-target": "https://example.com/slo",
+      "sso-target": "https://example.com/sso",
+      "attr-groups": "MemberOf",
+      "attr-site-admin": "SiteAdmin",
+      "site-admin-role": "site-admins",
+      "sso-api-token-session-timeout": 1209600
+    }
+  }
+}
+```
+
+## List SMTP Settings
+`GET /api/v2/admin/settings/smtp`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                         | Reason
+--------|--------------------------------------------------|-------
+[200][] | [JSON API document][] (`type: "settings/saml"`) | Successfully listed SMTP settings
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/admin/settings/smtp
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "smtp",
+    "type": "settings/smtp",
+    "attributes": {
+      "enabled": true,
+      "host": "example.com",
+      "port": 25,
+      "sender": "sample_user@example.com",
+      "auth": "login",
+      "username": "sample_user"
+    }
+  }
+}
+```
+
+## Update SMTP Settings
+`PATCH /api/v2/admin/settings/smtp`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
```

Comment:
FYI: in David's PR, he omitted these notes and instead called out the auth requirements once, on the index page. @armchairlinguist, you have more direct contact with our readers, and you've probably seen confusion about token types firsthand; do you reckon we should go back through the other admin pages later and make sure every endpoint has this note?

Reply:
I prefer the way David did it - there's a lot less duplication. If every admin endpoint requires a user token, we can say that once and point people back there if they have questions.

Reply:
So which way should I go? David's description of authentication appears to be in the future index file. I can just remove all references to authentication from this setting file if that's what you want?

Reply:
Yes, I think removing the references to authentication is the way to go. 👍🏼